### ISSUE TITLE
New parser request: swapon

### ISSUE THREAD

**Issue by don-rumata** [kellyjonbrazil/jc#383]
Hi, can you add support for `swapon` (--output-all|--raw|--show|--summary) output?

Examples:
1.

`LC_ALL=C swapon --output-all`
```
NAME      TYPE SIZE USED PRIO UUID                                 LABEL
/swap.img file   2G   2M   -2 0918d27e-3907-471d-abb8-45fa49ae059c 
```
`LC_ALL=C swapon --raw`
```
NAME TYPE SIZE USED PRIO
/swap.img file 2G 2M -2
```
`LC_ALL=C swapon --show`
```
NAME      TYPE SIZE USED PRIO
/swap.img file   2G   2M   -2
```
`LC_ALL=C swapon --summary`
```
Filename                                Type            Size    Used    Priority
/swap.img                               file            2097148 2080    -2
```

2.

`LC_ALL=C swapon --output-all`
```
NAME      TYPE      SIZE USED PRIO UUID                                 LABEL
/dev/sda5 partition   2G   0B   -2 5342ad64-eb35-47b9-a34f-84da56e06fce 
```
`LC_ALL=C swapon --raw`
```
NAME TYPE SIZE USED PRIO
/dev/sda5 partition 2G 0B -2
```
`LC_ALL=C swapon --show`
```
NAME      TYPE      SIZE USED PRIO
/dev/sda5 partition   2G   0B   -2
```
`LC_ALL=C swapon --summary`
```
Filename                                Type            Size    Used    Priority
/dev/sda5                               partition       2097148 0       -2
```

**Comment by kellyjonbrazil** [comment]
Thanks for the parser suggestion! It looks like the `asciitable` parser just might work for many of these but there are some formatting issues with the filename spilling into the other columns sometimes. It looks like this should be pretty easy to do.

**Comment by merval** [comment]
I'll see if I can assist here.

**Comment by muescha** [comment]
This issue can be closed as `implemented` 
parser is on master:
- Pull Request merged: #489 swapon parser
